templates/common/infra/bicep/core/host/functions-dotnet.bicep

```diff
@@ -0,0 +1,36 @@
+param environmentName string
```

Comment:
@puicchan - Do we have an arch diagram for this?

Reply:
No. I have created a new issue for Austin https://github.com/Azure/azure-dev/issues/800.